spring-cloud-aws-messaging/pom.xml

```diff
@@ -45,6 +45,11 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-messaging</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
@maciejwalkowiak why do we need this dependency here ? I try to keep the boot depdencies out from the modules (only for release 2.0, later I want to add them).

Reply:
Good point. Leftover from previous imolementation. I'll remove it soon.